javascript/javascript3/week1/giphyApp/gifsApi.js

```diff
@@ -0,0 +1,74 @@
+//Giphy api
+// Create a site where a user can search for any word using gif api
+//https://developers.giphy.com/docs/ 
+//my api id= Q3MkdtrQmN4KgeUL7fqzb5vn1Bjt628g
+
+ 
+const getGifs=()=>{
+    const inputSearching=document.getElementById('inputSearch');
+    
+    let inputWord=inputSearching.value;
+
+    if(inputWord!==''){
+        let url=`https://api.giphy.com/v1/gifs/search?&api_key=Q3MkdtrQmN4KgeUL7fqzb5vn1Bjt628g&q=${inputWord}`;
+
+        fetch(url)
+        .then(res=>res.json())
+        .then(result=>{
+            console.log(result)
+            
+            const div=document.getElementById("gif");
+            const dataArr=result.data;
```

Comment:
`dataArr` name is too general, try to not use `arr`, or `array` in variable names